cmd/podman/network.go

```diff
@@ -19,6 +19,7 @@ var networkcheckCommand = cliconfig.PodmanCommand{
 
 // Commands that are universally implemented
 var networkcheckCommands = []*cobra.Command{
+	_networkcreateCommand,
```

Comment:
Wait, if this is in universally-implemented commands - why is `network_create.go` set as `+build !remoteclient`

Reply:
removed comment